.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,81 @@
+name: Container for Kanidm
+
+# this will build regardless, but only push to the container registry when you're committing on the master branch.
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: icepuma/rust-action@master
+        with:
+          args: cargo test
+
+  kanidm_build:
+    needs: test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+         - linux/arm64
```

Comment:
Which image tags is this building?

Reply:
These are mentioned later:

`tags: ghcr.io/${{ github.repository }}/kanidmd:latest`

and 

`tags: ghcr.io/${{ github.repository }}/radius:latest`

from previous testing it *should* end up as `kanidm/radius:latest` etc, we could rename it as `dev` or the less... accurate `nightly` which seems to match people's usual expectations.

This incarnation's more about having a testable container available on the github container registry than a "this is the current stable one" which I'd probably push to dockerhub or something?

Reply:
Having said that, we could also have another github actions config to build the tagged versions to match the releases....

Reply:
I think latest is our release tag name, we should call this devel i think. Check Makefile i think it has the name.

Reply:
Oh hang on ghcr.io is a github registry, and this is not pushed to docker hub correct?

Reply:
Anyway calling it devel keeps it inline with the docker hub images

Reply:
Yep, devel's the one, missed that in all the Makefile templateyness. Sorted.